contrib/envoy/bootstrap.yaml

```diff
@@ -27,7 +27,7 @@ static_resources:
             endpoint:
               address:
                 socket_address:
-                  address: "localhost"
+                  address: BOOTSTRAP_ADDRESS
```

Comment:
This file could be used by others, so instead of modifying it, can you copy it and add an extension that makes it look like a "templated file" (so, it's explicit that is not intended to be used directly by envoy) something like `bootstrap.yaml.tmpl` or so.

Reply:
Sure, added in 0e7af08b3b7783824fb7f6c7bb25f553d22b36dc.